department-of-veterans-affairs/va.gov-cms
# VACMS-3142: Update toolbar perms and phpunit test.

## Description
See #3142 

## Testing done
Visual, PhpUnit

## QA steps
Login as a content editor, and visually verify that 
![image](https://user-images.githubusercontent.com/2404547/95678722-7921e400-0b9c-11eb-8cf6-b4bb3b1653fe.png)
 is visible in toolbar
